### PR TITLE
Chore/adjust cellar functions

### PR DESCRIPTION
**Pull request by codingki**
## Description

This PR is for adjusting cellar functions, move the same contract functions and calculations to `src/data/actions/common/`

## Changes

- [x] [chore: adjust active asset functions](https://github.com/strangelove-ventures/sommelier/commit/0954db56988766448a5425c72d79f922ddba5fa5)
- [x] [chore: add clear gate apy function](https://github.com/strangelove-ventures/sommelier/commit/35f5eefc74f72b1383e9e85857d74ea93c010d8b)
- [x] [chore: update hardcoded content](https://github.com/strangelove-ventures/sommelier/commit/499465f4389494d4e352ad1efdfd2e4de4d9ccc0)

### CODE REVIEW

File: src/data/actions/CLEAR_GATE_CELLAR/getApy.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const getApy = () => {
+  // the value is pulled from cellarDataMap.overrideApy
+  return null
```
Comment (xanderjl) [inline src/data/actions/CLEAR_GATE_CELLAR/getApy.ts:3]: If this is being overridden elsewhere, do we need this function? Couldn't we have some sort of optional chaining that makes a call to the common `getApy` if the value isn't being pulled in from the `cellarDataMap`?

File: src/data/hooks/useApy.ts [hunk 2]
```diff
@@ -23,10 +27,15 @@ export const useApy = (config: ConfigProps) => {
           stakerContract as SommStaking
         )
       }
+      if (config.cellar.key === CellarKey.CLEAR_GATE_CELLAR) {
+        return getApy_CLEAR_GATE()
```
Comment (xanderjl) [inline src/data/hooks/useApy.ts:31]: See comment above re: this pattern

**Review by xanderjl — CHANGES_REQUESTED** [review]
Looks great! Only question is around that `getApy` that returns `null`. Wonder if we can handle that in the hook instead? I feel like I'm missing context around why this call is necessary

File: src/data/actions/CLEAR_GATE_CELLAR/getApy.ts (hunk 1 above)
Comment (codingki) [reply src/data/actions/CLEAR_GATE_CELLAR/getApy.ts:3]: We don't really need this function. Yes We could. Good point, Thanks!

**Review by xanderjl — APPROVED** [review]
Sweet! Looks great @codingki

**Pull request merged**